recipes/pybind11/all/test_package/CMakeLists.txt

```diff
@@ -1,16 +1,14 @@
 cmake_minimum_required(VERSION 2.8.11)
 project(test_package CXX)
 
-set(CMAKE_VERBOSE_MAKEFILE TRUE)
-
-find_package(PythonInterp REQUIRED)
-find_package(PythonLibs REQUIRED)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_RELEASE ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY_RELEASE ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY_DEBUG ${CMAKE_RUNTIME_OUTPUT_DIRECTORY})
 
 find_package(pybind11 REQUIRED)
-include(pybind11Tools)
-
-include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
-conan_basic_setup()
```

Comment:
Isn't the cmake helper always required to make sure that the compiler flags are set correctly?

Reply:
It is not mandatory. Here I wanted to test that the package can be consumed ina transparent way with the CMake macros they provide.

Reply:
It will be needed when a cpython recipe is used.
Can't the call to `conan_basic_setup` be moved to after the `find_package`?

Reply:
Yes, please, leave the `conan_basic_setup()` call, it is our _toolchain_ until Conan provides a proper toolchain (WIP).